dataproc/dataproc_test.go

```diff
@@ -63,7 +63,7 @@ func TestDataproc(t *testing.T) {
 
 	buf := new(bytes.Buffer)
 
-	testutil.Retry(t, 3, 30*time.Second, func(r *testutil.R) {
```

Comment:
Retries aren't working well. So, let's give UUIDs a try.

Reply:
Done. I did add retries to the workflow sample, as I can't easily slot UUID into the sample without modifying it, which I would like to avoid if possible.